webui/react/src/pages/ExperimentDetails/ExperimentSingleTrialTabs.tsx

```diff
@@ -249,8 +251,13 @@ const ExperimentSingleTrialTabs: React.FC<Props> = ({
         ) : (
           <TrialDetailsHyperparameters pageRef={pageRef} trial={trialDetails as TrialDetails} />
         ),
+        disabled: experiment.unmanaged,
         key: TabType.Hyperparameters,
-        label: 'Hyperparameters',
+        label: experiment.unmanaged ? (
+          <Tooltip content={UNMANAGED_MESSAGE}>Hyperparameters</Tooltip>
+        ) : (
+          'Hyperparameters'
+        ),
```

Comment:
I don't think we want to hide the entire `Hyperparameters` tab, only `Hyperparameters Search` is not supported

Reply:
ok. I misunderstood the ticket description.  Will fix it